migrations/20180212145410-add-external-id.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * Copyright (c) 2017, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or
+ * https://opensource.org/licenses/BSD-3-Clause
+ */
+
+'use strict'; // eslint-disable-line strict
+
+module.exports = {
+  up: function (qi, Sequelize) {
```

Comment:
don't need to do it here but when there's more than one step in the migration, please wrap it in a transaction